modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc

```diff
@@ -0,0 +1,136 @@
+//do we need to have installed kernel-rt on the host before we can make the blueprint?
+* You are using the x86_64 architecture.
+//any other prereqs?
+
+[NOTE]
+====
+The example blueprint includes references gathered from the previous steps required to configure low latency for a{microshift-short} cluster.
+====
+
+[IMPORTANT]
+====
+A subscription that includes the {op-system-rtk} must be enabled on the host used to build the commit.
+====
+
+.Procedure
+
+. Use the following example blueprint to install the real-time kernel in a {op-system-ostree} image:
++
+[source,text]
+----
+name = "microshift-low-latency"
+description = "RHEL 9.4 and MicroShift configured for low latency"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
+
+[[packages]]
+name = "microshift"
+version = "*"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "*"
+
+[[packages]]
+name = "microshift-networking"
+version = "*"
+
+[[packages]]
+name = "microshift-selinux"
+version = "*"
+
+[[packages]]
+name = "microshift-low-latency"
+version = "*"
+
+[[packages]]
+name = "systemd-resolved"
+version = "*"
+
+# Kernel RT is only available for x86_64
+[customizations.kernel]
+name = "kernel-rt"
+
+[customizations.services]
+enabled = ["microshift", "microshift-tuned"]
+
+[customizations.firewall]
+ports = ["22:tcp", "80:tcp", "443:tcp", "5353:udp", "6443:tcp", "30000-32767:tcp", "30000-32767:udp"]
+
+[customizations.firewall.services]
+enabled = ["mdns", "ssh", "http", "https"]
+
+[[customizations.firewall.zones]]
+name = "trusted"
+sources = ["10.42.0.0/16", "169.254.169.1"]
+
+[[customizations.files]]
+path = "/etc/microshift/config.yaml"
+data = """
+kubelet:
+  cpuManagerPolicy: static
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true"
+  cpuManagerReconcilePeriod: 5s
+  memoryManagerPolicy: Static
+  topologyManagerPolicy: single-numa-node
+  reservedSystemCPUs: 0-1
+  reservedMemory:
+  - limits:
+      memory: 1100Mi
+    numaNode: 0
+  kubeReserved:
+    memory: 500Mi
+  systemReserved:
+    memory: 500Mi
+  evictionHard:
+    imagefs.available: 15%
+    memory.available: 100Mi
+    nodefs.available: 10%
+    nodefs.inodesFree: 5%
+  evictionPressureTransitionPeriod: 0s
+"""
+
+[[customizations.files]]
+path = "/etc/tuned/microshift-baseline-variables.conf"
+data = """
+# Isolated cores should be complementary to kubelet's reserved CPUs.
+# Isolated and reserved CPUs should contain all online CPUs.
+# Core #3 is for testing offlining hence skipped.
+isolated_cores=2,4-5
+hugepages_size=2M
+hugepages=10
+additional_args=test1=on test2=true dummy
+offline_cpu_set=3
+"""
+
+[[customizations.files]]
+path = "/etc/microshift/tuned.yaml"
+data = """
+profile: microshift-baseline
+reboot_after_apply: True
+"""
+----
+//TODO add some footnotes to blueprint?
+
+. Restart the host to activate the real-time kernel.
+//Is there anything on restart that tells us the kernel is active?
```

Comment:
We can use `cat /proc/cmdline` to get path to currently booted kernel which should end with `+rt`